make/upgrade

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Upgrade a running CF using the current chart in output/helm. Assumes
+# that only one revision is currently running. If more than one runs
+# the last per helm's sorting order is chosen. (See **)
+
+set -o errexit -o nounset
+
+NAMESPACE="cf"
+UAA_NAMESPACE="uaa"
+UAA_CA_CERT=internal-ca-cert
+
+GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
+cd "${GIT_ROOT}"
+
+# Determine the kube revision of the chart controlling the specified namespace.
+release_version() { helm list --namespace "$1" | awk '{ print $2 }' | tail -n 1 ; }
+
+get_uaa_secret () {
+    kubectl get secret secret-$(release_version "${UAA_NAMESPACE}") \
+	--namespace "${UAA_NAMESPACE}" \
+	-o jsonpath="{.data['$1']}"
+}
+
+# Argument? (**)
+RELEASE=$(helm list --namespace "${NAMESPACE}" | awk '{ print $1 }' | tail -n 1)
+echo Upgrading ${NAMESPACE} release \"${RELEASE}\" ...
+#exit
+
+stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-run start
+stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-run::upgrade start
+
+DOMAIN=${DOMAIN:-}
+NETWORK_ENV=bin/settings/network.env
+if test -n "${DOMAIN}"; then
+    TMP=$(mktemp -d)
```

Comment:
Why isn't it just a file? Why make it a directory?

I _think_ if we tried harder we could make this not have a temporary file.

Reply:
This part of the script is inherited from `make/run`. I saw no reason to fiddle with parts which worked.
Not changing.